lib/confirmation.rb

```diff
@@ -0,0 +1,30 @@
+require 'dotenv/load'
```

Comment:
This sounds like a really interesting way to set up your environmental variables for twilio, I'd be interested in hearing more about it :)